app/lib/liquid/template_support.rb

```diff
@@ -74,29 +76,24 @@ def render_template(context, options)
     end
 
     def assigns_for_liquify
-      original = {}
-
       @_assigned_drops ||= {}
       @_template_assigns ||= {}
 
       report_and_supress_exceptions do
-        diff = original.keys - @_assigned_drops.keys
         overriden = @_template_assigns.keys & @_assigned_drops.keys
 
-        if diff.present?
```

Comment:
why removing that?

Reply:
this code is never executed. diff is always empty